runtime/wasm/src/host.rs

```diff
@@ -307,22 +307,24 @@ impl RuntimeHost {
             .cloned()
             .collect::<Vec<_>>();
 
-        if handlers.is_empty() {
-            bail!(
+        ensure!(
+            !handlers.is_empty(),
+            format!(
```

Comment:
There is no need to trigger another round of review on account of this, but `ensure!` already supports formatting so there is no need for the `format!` here.